lib/ActionSheetCustom.js

```diff
@@ -24,7 +24,7 @@ class ActionSheet extends React.Component {
     }
   }
 
-  componentWillReceiveProps (nextProps) {
+  UNSAFE_componentWillReceiveProps (nextProps) {
```

Comment:
@falcon027 Why not replace the unsafe lifecycle with the new one `getSnapshotBeforeUpdate`?
I guess it is applicable in this place.
Using unsafe one will require additional changes in the future in this area